src/main/resources/fabric.mod.json

```diff
@@ -39,6 +39,9 @@
 	"team_reborn_energy": ">=2.0.0-beta1",
 	"fabric-biome-api-v1": ">=3.0.0"
   },
+  "mixins": [
+	"techreborn.mixins.json"
```

Comment:
Ideally these mixins should go into RebornCore (Just a sub project), we try to keep TR free of mixins. Should hopefully not be too hard to do this, if you are unsure and need some pointers please let me know. 👍 

Otherwise it looks resonable at a glance.

Reply:
Thanks!
I moved changes to RebornCore (and formatted mixins files a bit for correct automatic line sorting)